Zend/zend_weakrefs.c

```diff
@@ -158,7 +158,7 @@ ZEND_METHOD(WeakReference, create)
 {
 	zval *referent;
 
-	ZEND_PARSE_PARAMETERS_START_EX(ZEND_PARSE_PARAMS_THROW, 1,1)
+	ZEND_PARSE_PARAMETERS_START(1,1)
```

Comment:
I don't think this change is right.

cc @krakjoe

Reply:
I was in the belief that this flag is not needed anymore because:

`#define ZEND_PARSE_PARAMS_THROW 0 /* No longer used, zpp always uses exceptions */`

Reply:
Yup, no longer needed.

Reply:
@kocsismate In that case, feel free to open a PR removing the usages, as well https://github.com/php/php-src/pull/4875#discussion_r340627334.

Reply:
@carusogabriel I've already checked the usages: this was the last one. I was thinking about the possibility to remove the definition itself (in a separate PR), but I wasn't sure if it is worth to make such a BC change in the API at this point. If it is, I can create a PR for that.

Reply:
Yep, that was my guess so that's why I didn't want to do anything with it.

Reply:
@nikic, could we define zpp_throw as macro?

Reply:
Yeah, as long as there's no prob with variadic macros. We should at least remove all the usages that are still left...